src/prefect/tasks/aws/lambda_function.py

```diff
@@ -0,0 +1,407 @@
+from base64 import b64encode
+import json
+
+import boto3
+
+from prefect import Task
+from prefect.client import Secret
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class LambdaCreate(Task):
+    """
+    Task for creating a Lambda function.
+
+    Args:
+        - function_name (str): name of teh Lambda function to create
```

Comment:
```suggestion
        - function_name (str): name of the Lambda function to create
```

Reply:
Corrected locally and pushed, good catch